src/test/java/Homework19.java

```diff
@@ -0,0 +1,55 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.WebElement;
+
+    public class Homework19 extends BaseTest {
+
+@Test
+public void deletePlaylist() throws InterruptedException {
+    setupClass();
+
+    navigateToPage();
+    provideEmail("test@test.com");
+    providePassword("te$t$tudent");
+    clickSubmit();
+    choosePlaylist();
+    clickDeletePlaylistBtn();
+    Assert.assertTrue(confirmNotification().contains("Deleted playlist"));
+}
+
+
+        public void choosePlaylist() throws InterruptedException {
+    WebElement clickPlaylist = driver.findElement(By.xpath("//*[@id='playlists']/ul/li[3]/a"));
+    clickPlaylist.click();
+    Thread.sleep(1000);
+}
+        public void clickDeletePlaylistBtn(){
+        WebElement deletePlaylistClick = driver.findElement
+                (By.xpath("//div[@class='song-list-controls']//button[@class='del btn-delete-playlist']"));
+        deletePlaylistClick.click();
+        }
+
+        public String confirmNotification() {
+            WebElement messageDelete = driver.findElement(By.cssSelector("div.success.show"));
+            return messageDelete.getText();
+        }
+
+  }
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._